src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="/src/styles/main.scss"
```

Comment:
The stylesheet link references a `.scss` file. Typically, `.scss` files are preprocessed into `.css` files. Ensure that the SASS file is being compiled into a CSS file and that the correct path is referenced here.